orchestra/project_api/tests/test_project_api.py

```diff
@@ -630,32 +610,3 @@ def test_destroy_functionality(self):
         marked_as_deleted = Todo.unsafe_objects.get(pk=self.todo.pk)
         self.assertTrue(marked_as_deleted.is_deleted)
         self.assertEqual(marked_as_deleted, self.todo)
-
-    def test_bulk_update(self):
```

Comment:
In this PR, we removed update and retrieve, but not delete and create. i wonder if we should just leave these tests in this PR as-is untouched?